Patches/MechanoidsExtraordinaire/ThingDefs_Races/Races_Mechanoid_MechanoidsExtraordinaire.xml

```diff
@@ -179,7 +184,8 @@
 							<power>10</power>
 							<cooldownTime>1.45</cooldownTime>
 							<linkedBodyPartsGroup>HeadAttackTool</linkedBodyPartsGroup>
```

Comment:
HeadAttackTool but labeled as "right fist"?

Reply:
This belongs to the Chimera Mechanoid, which according to the original mod's description, is deliberately made up of a hodgepodge of Mechanoid remains, hence the bizarre mismatch between the label and linkedBodyPartGroup:

> "This strange type of a mechanoid was said to have been made out of scraps of deceased members of its own race. To defend themselves, chimeras use powerful inferno cannons as well as blades, which used to belong to scythers, installed on the tips of their curled tails."

Reply:
Hmm... I'll take a look through this at some point and try to make sense of what the changes are. I'm not familiar with the mod, but a lot of this stuff just doesn't make a lot of sense to me.